<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="SFMLCanvas" name="sfml" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>501</width>
      <height>471</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="plotRT" native="true">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>0</y>
      <width>841</width>
      <height>321</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>490</y>
      <width>488</width>
      <height>177</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="2" column="1">
      <widget class="QLineEdit" name="scale">
       <property name="text">
        <string>3</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;начать рассчет задачи&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="start">
         <property name="text">
          <string>начать</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="method">
       <item>
        <property name="text">
         <string>метод Эйлера</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>метод Рунге-Кутте</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="1">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="0" rowspan="5">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;начальные условия&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="height_">
             <property name="text">
              <string>высота орбиты в метрах                       </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="height">
             <property name="text">
              <string>7.4e6</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="angle_">
             <property name="text">
              <string>начальный угол (в радианах)                </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="angle">
             <property name="text">
              <string>0.3</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="velocity_">
             <property name="text">
              <string>начальная скорость(в м/c)                    </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="velocity">
             <property name="text">
              <string>4000</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="angle_2_">
             <property name="text">
              <string>погрешность рассчета(в м)                   </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="error">
             <property name="text">
              <string>3e5</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="angle_velocity_">
             <property name="text">
              <string>начальная угловая скорость(в рад/c)    </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="angle_velocity">
             <property name="text">
              <string>1.14e-3</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;масштаб&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QCustomPlot" name="plotPHIT" native="true">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>330</y>
      <width>841</width>
      <height>351</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1366</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>SFMLCanvas</class>
   <extends>QWidget</extends>
   <header>sfmlcanvas.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
